include/seqan3/alphabet/aminoacid/aa27.hpp

```diff
@@ -31,28 +31,33 @@
 // DAMAGE.
 //
 // ============================================================================
-// Author: Sara Hetzel <sara.hetzel AT fu-berlin.de>
-// ============================================================================
+
+/*!\file aa27.hpp
+ * \ingroup aminoacid
+ * \author Sara Hetzel <sara.hetzel AT fu-berlin.de>
+ * \brief Contains seqan3::aa27, container aliases and string literals.
+ */
 
 #pragma once
 
 #include <cassert>
 
 #include <seqan3/alphabet/concept.hpp>
```

Comment:
The test needs the header, but the test already includes it, by including the alphabet meta header.
This *should* work, but if it doesn't you can try changing the include order in the alphabet meta header...
In any case the definition of aa27 should not depend on the concept (OTOH the container includes do need to be there)...